src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -292,15 +292,20 @@ public IEnumerable<PackageItemViewModel> GetCurrent()
                     Sources = _packageSources,
                     PackagePath = metadata.PackagePath,
                     PackageFileService = _packageFileService,
-                    IncludePrerelease = _includePrerelease
+                    IncludePrerelease = _includePrerelease,
                 };
 
+                if (metadata.TransitiveOrigins != null)
+                {
+                    listItem.PackageLevel = metadata.TransitiveOrigins.Any() ? PackageLevel.Transitive : PackageLevel.TopLevel;
```

Comment:
Isn't TransitiveOrigins set for transitve packages only?

Reply:
Yes. That's why that list could be null for top-level, installed packages.

Reply:
So no need to check for TransitiveOrigins.Any() then.